build.gradle

```diff
@@ -1,76 +1,56 @@
-/*
- Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
- http://www.apache.org/licenses/LICENSE-2.0
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
- */
-
-plugins {
-    id 'java-library'
-    id 'application'
-}
-
-group = 'com.exactpro.th2'
-version = release_version
-
-repositories {
-    mavenCentral()
-    maven {
-        name 'Sonatype_snapshots'
-        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+buildscript {
+    repositories {
+        gradlePluginPortal()
+        maven {
+            url = "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+        }
     }
-    maven {
-        name 'Sonatype_releases'
-        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+    dependencies {
+        classpath "com.exactpro.th2:th2-gradle-plugin:0.0.1-dev-5915772757-13a28ae-SNAPSHOT"
     }
-    flatDir {
-        dirs "lib"
-    }
-    mavenCentral()
 }
 
-dependencies {
-    api platform('com.exactpro.th2:bom:3.0.0')
+apply plugin: "com.exactpro.th2.common-conventions"
+apply plugin: "com.exactpro.th2.docker-conventions"
 
-    compile 'com.exactpro.th2:common:3.13.4'
-    compile "com.exactpro.sf:sailfish-common:3.2.1603"
+apply plugin: 'kotlin-kapt'
+apply plugin: 'antlr'
```

Comment:
I don't think we need the antlr plugin here. Could you please remove it from build.gradle?